app/serializers/v2/supported_profile_serializer.rb

```diff
@@ -3,6 +3,12 @@
 module V2
   # JSON serialization for SupportedProfile
   class SupportedProfileSerializer < V2::ApplicationSerializer
-    attributes :title, :ref_id, :security_guide_id, :security_guide_version, :os_major_version, :os_minor_versions
+    attributes :title,
+               :description,
+               :ref_id,
+               :security_guide_id,
+               :security_guide_version,
+               :os_major_version,
+               :os_minor_versions
```

Comment:
We have multiline attributes specified already in other serializers, check their syntax and try to keep it consistent.